bootstrap/kubeadm/types/v1beta1/conversion.go

```diff
@@ -68,3 +69,9 @@ func (dst *JoinConfiguration) ConvertFrom(srcRaw conversion.Hub) error {
 	src := srcRaw.(*bootstrapv1.JoinConfiguration)
 	return Convert_v1alpha4_JoinConfiguration_To_v1beta1_JoinConfiguration(src, dst, nil)
 }
+
+func Convert_v1beta1_DNS_To_v1alpha4_DNS(in *DNS, out *bootstrapv1.DNS, s apimachineryconversion.Scope) error {
+	out.ImageRepository = in.ImageRepository
+	out.ImageTag = in.ImageTag
```

Comment:
This should not be necessary, because conversion for `ImageMeta` should be managed by `autoConvert_v1beta1_DNS_To_v1alpha4_DNS`.
Instead could you please add a the following note:
```
// DNS.Type does was removed in v1alpha4 because only CoreDNS is supported, dropping this info.
```

Reply:
Right, I didn't notice it was an embedded struct!